src/wandbot/evaluation/eval/async_main.py

```diff
@@ -200,15 +208,18 @@ def log_eval_result(eval_result_path: str, duration: float) -> None:
     final_eval_results.update(percent_grade1.to_dict())
     final_eval_results.update(percentage_true_results.to_dict())
 
-
-    logger.info(f"Final Eval Results: {json.dumps(final_eval_results, indent=4)}")
+    logger.info(
+        f"Final Eval Results: {json.dumps(final_eval_results, indent=4)}"
+    )
     run.log(final_eval_results)
 
     run.summary["duration(s)"] = duration
 
 
 async def main():
-    eval_artifact = wandb.Api().artifact("wandbot/wandbot-eval/autoeval_dataset:v3")
+    eval_artifact = wandb.Api().artifact(
```

Comment:
can this be made to pull from a config instead?

Reply:
Updated to use config